.github/workflows/publish-autodoc.yml

```
@@ -0,0 +1,52 @@
+name: publish autodoc
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: false
+        description: "version override"
+        type: string
+
+jobs:
+  generate-and-deploy-doc:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: eclipse-edc/.github/.github/actions/setup-build@main
+
+      - name: Override version if input is set
+        if: "${{ github.event.inputs.version != '' }}"
+        run: sed -i "s/version=.*/version=${{ github.event.inputs.version }}/g" gradle.properties
+
+      - name: Generate docs
+        run: ./gradlew autodoc
+      - name: Merge manifests
+        run: ./gradlew mergeManifest
```

```suggestion
        run: ./gradlew mergeManifests
```
(should not make a difference though)